### PR TITLE
feat(Toolbar): Add toolbarBehavior and toolbarItemBehavior

### LINKED ISSUE #1413: Toolbar - accessibility behavior
Folloup of #1408.

Add behaviors for `Toolbar`, `ToolbarItem` and `ToolbarRadiogroup`.

WAI-ARIA spec: https://www.w3.org/TR/wai-aria-practices/examples/toolbar/toolbar.html

### PR DESCRIPTION
**Pull request by sophieH29**
closes https://github.com/stardust-ui/react/issues/1413

- improved `buttonBehavior`
- improved `toggleButtonBehavior`
- added `toolbarItemBehavior`

### CODE REVIEW

File: packages/react/src/components/Toolbar/ToolbarItem.tsx [hunk 1]
```diff
@@ -25,6 +26,7 @@ export interface ToolbarItemProps
     ContentComponentProps {
   /**
    * Accessibility behavior if overridden by the user.
+   * @default toolbarItemBehavior
```
Comment (silviuaavram) [inline packages/react/src/components/Toolbar/ToolbarItem.tsx]: wasn't this supposed to have the same value as the one in `defaultProps`?

File: packages/react/src/lib/accessibility/Behaviors/Button/toggleButtonBehavior.ts [hunk 2]
```diff
@@ -1,23 +1,24 @@
...
-      'aria-disabled': !_.isNil(props['aria-disabled'])
-        ? props['aria-disabled']
-        : !!props['disabled'],
-      'aria-pressed': !_.isNil(props['aria-presssed']) ? props['aria-presssed'] : !!props['active'],
-    },
-  },
-})
+const toggleButtonBehavior: Accessibility = (props: any) => {
+  const behaviorData = buttonBehavior(props)
+  console.log('toggleButtonBehavior', !!props['active'])
```
Comment (silviuaavram) [inline packages/react/src/lib/accessibility/Behaviors/Button/toggleButtonBehavior.ts]: don't forget about this.

**Review by silviuaavram — APPROVED** [review]

File: packages/react/src/components/Toolbar/ToolbarItem.tsx (hunk 1 above)
Comment (sophieH29) [reply packages/react/src/components/Toolbar/ToolbarItem.tsx]: yup, you're right

File: packages/react/src/lib/accessibility/Behaviors/Button/toggleButtonBehavior.ts (hunk 2 above)
Comment (sophieH29) [reply packages/react/src/lib/accessibility/Behaviors/Button/toggleButtonBehavior.ts]: 😃 thank you!

**Review by miroslavstastny — APPROVED** [review]

**Review by jurokapsiar — APPROVED** [review]

File: packages/react/src/components/Toolbar/ToolbarItem.tsx [hunk 3]
```diff
@@ -83,14 +85,22 @@ class ToolbarItem extends UIComponent<WithAsProp<ToolbarItemProps>, ToolbarItemS
 
   static defaultProps = {
     as: 'button',
-    accessibility: defaultBehavior as Accessibility,
+    accessibility: buttonBehavior as Accessibility,
+  }
+
+  actionHandlers = {
+    performClick: event => {
+      event.preventDefault()
```
Comment (jurokapsiar) [inline packages/react/src/components/Toolbar/ToolbarItem.tsx:116]: to make link button work:
```
if(this.props.as !=== 'a') {
  event.preventDefault()
}
```

**Pull request merged**